app/konnect/api-products/service-documentation.md

```diff
@@ -24,9 +18,25 @@ You can provide extended descriptions of your API products with a Markdown (`.md
   <img src="/assets/images/products/konnect/api-products/konnect_service_docs_description.png" />
 </p>
 
-All API product descriptions are managed from the **Documentation** section in the API Product overview. Once you've uploaded the markdown file, you have a preview of how it will render, the option to edit, and a view of the publication status. You can also create a hierarchy between the docs you upload that will be reflected in the way they're displayed in the Dev Portal.
 
-## API specification
+Manage all API product descriptions from the Documentation section in the API Product overview. After uploading your markdown file, you'll be able to preview its rendering, edit it directly using the {{site.konnect_saas}} Markdown Renderer, and check its publication status. Additionally, you can organize your documents into a hierarchy, which will influence the order they are displayed to your users from within the Dev Portal.
+
+### Interactive markdown renderer
+
+The integrated markdown editor allows you to edit your documentation directly within {{site.konnect_saas}}. It supports:
+
+* Code syntax highlighting for Bash, JSON, Go, and JavaScript
+* Support for rendering UML diagrams and flowcharts via Mermaid and PlantUML
+* Support for emojis
+
+In addition to the typical Markdown support for code syntax highlighting, Mermaid and PlantUML diagrams are supported by using a language-specific identifier immediately following the triple backtick (```) notation that initiates the code block, for example: 
```

Comment:
```suggestion
You can insert Mermaid and PlantUML diagrams by using a language-specific identifier immediately following the triple backtick (```) notation that initiates the code block:
```

Since we already mentioned that these two are supported, could we rephrase this paragraph so it's more so about how to include these in your docs instead of that they are supported?